package.json

```diff
@@ -73,16 +73,16 @@
     "graphql": "15.8.0",
     "husky": "7.0.4",
     "prettier": "2.6.2",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
+    "react": "17.0.1",
```

Comment:
I think these react changes were on accident, right? We should be able to just checkout the package.json file from main once apollo-client is not a dependency

Reply:
Lets make sure the package lock hasn't changed before we merge 🙏